lib/portlayer/storage/vsphere/image.go

```diff
@@ -288,14 +288,14 @@ func (v *ImageStore) Export(op trace.Operation, store *url.URL, id, ancestor str
 	mounts := []*object.DatastorePath{}
 	cleanFunc := func() {
 		for _, mount := range mounts {
-			if err := v.dm.UnmountAndDetach(op, mount); err != nil {
+			if err := v.dm.UnmountAndDetach(op, mount, !persistent); err != nil {
```

Comment:
@caglar10ur what would it take to pass a *VirtualDisk here instead? Doing it this way makes us dependent on knowing the persistence value that was used when mounting, however that might not be as simple as in this scenario.